### PR TITLE
feat(javascript): add batchSize to the observability event object

### PR DESCRIPTION
**Pull request by Fluf22**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-1521

### Changes included:

- add batchSize to the observability event object

### CODE REVIEW

File: specs/ingestion/common/schemas/event.yml [hunk 1]
```diff
@@ -25,6 +31,7 @@ Event:
     - runID
     - status
     - type
+    - batchSize
```
Comment (shortcuts) [inline specs/ingestion/common/schemas/event.yml:34]:
```suggestion
```

it's not required we will default it

**Review by shortcuts — COMMENTED** [review]
thanks!

File: specs/ingestion/common/schemas/event.yml (hunk 1 above)
Comment (millotp) [reply specs/ingestion/common/schemas/event.yml:34]: yes it's required that the API returns it

Comment (shortcuts) [reply specs/ingestion/common/schemas/event.yml:34]: ah true it's not the insert here gg

Comment (Fluf22) [reply specs/ingestion/common/schemas/event.yml:34]: Is it just me or is there an indentation issue? 🤔

Comment (shortcuts) [reply specs/ingestion/common/schemas/event.yml:34]: seems good on my side!

Comment (Fluf22) [reply specs/ingestion/common/schemas/event.yml:34]: image

Comment (millotp) [reply specs/ingestion/common/schemas/event.yml:34]: tabs vs spaces on iphone is not the same size as on mac ?

Comment (Fluf22) [reply specs/ingestion/common/schemas/event.yml:34]: Ah maybe! As long as it's good in reality 👌🏻

**Review by shortcuts — DISMISSED** [review]
gg!

File: specs/ingestion/common/schemas/event.yml [hunk 2]
```diff
@@ -15,6 +15,12 @@ Event:
       $ref: '#/EventStatus'
     type:
       $ref: '#/EventType'
+    batchSize:
```
Comment (shortcuts) [inline specs/ingestion/common/schemas/event.yml:18]: ah we maybe need to say it's an integer, dart generates a double

File: specs/ingestion/common/schemas/event.yml [hunk 3]
```diff
@@ -15,6 +15,12 @@ Event:
...
+      type: number
```
Comment (shortcuts) [inline specs/ingestion/common/schemas/event.yml:19]:
```suggestion
      type: integer
```

should do it

File: specs/ingestion/common/schemas/event.yml (hunk 2 above)
Comment (Fluf22) [reply specs/ingestion/common/schemas/event.yml:18]: Ah, I'm always lost between number types.
That's why I put some validation, but it's better to use `integer` you're right.

**Review by shortcuts — APPROVED** [review]
lovely! gg

**Pull request merged**